persistence/proto/src/main/scala/net/liftweb/proto/ProtoUser.scala

```diff
@@ -864,8 +864,10 @@ trait ProtoUser {
       }
     }
 
+    val emailElemId = nextFuncName
+    S.appendJs(Focus(emailElemId))
     val bind =
-      ".email" #> FocusOnLoad(<input type="text" name="username"/>) &
+      ".email" #> <input type="text" name="username" id={emailElemId}/> &
```

Comment:
Wow… Yeah, you're incredibly right, I don't know what I was thinking O.O

Well with that in mind, I think this is fine to go in as is and we can always
do another PR to improve how we actually bind the fields, yeah?